### PR TITLE
RM url path

### PR DESCRIPTION
**Pull request by SamGodwin2**
### What is the context of this PR?
Changed the url for the RM service and updated the readme

### How to review 
New url is correct

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ Your app is ready to be deployed!
 
 See the section about [deployment](#deployment) for more information.
 
+# Enviroment Variables
+
+`REACT_APP_API_ROOT_URL` refers to the base url the RM service is being hosted on. Defaults to `http://localhost:8084/`
+
+`REACT_APP_UUID` refers to the unique identifyer RM uses to identify each excercise. This variable has no default and must be provided on app startup.
```
Comment (AdamHawtin) [inline README.md]: Can we rename this to something more specific like `REACT_APP_COLLECTION_EXERCISE_ID`?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MebinAbraham) [reply README.md]: Why does it need to be prefixed with `REACT_APP`, is that a bp?

Comment (SamGodwin2) [reply README.md]: Its a security  thing all react envs have to be prefixed with `REACT_APP` helps prevent private envs from being exposed.

Comment (MebinAbraham) [reply README.md]: Just change the UUID bit as a lot of things are UUID's, explicitly stating makes it clearer. 👍

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**